torch/csrc/tensor/python_tensor.cpp

```diff
@@ -78,6 +78,11 @@ static PyObject* Tensor_new(
   if (tensor_type.is_cuda && !torch::utils::cuda_enabled()) {
     throw unavailable_type(tensor_type);
   }
+  if (tensor_type.is_cuda) {
```

Comment:
I don't recall exactly the context in which this function gets called. Is this only ever called via the class constructor and type here will only be is_cuda for these torch.cuda classes? Can other subclasses or calling this with other arguments end up here as well?

Reply:
I've already done the tests, and this alarm will not appear when using `torch.tensor`

Reply:
Awesome! Thanks for checking.